main/webapp/modules/core/styles/views/data-table-view.less

```diff
@@ -356,3 +356,16 @@ ul.sorting-dialog-blank-error-positions > li {
   cursor: move;
   .rounded_corners(3px);
   }
+
+.interfaceForTextArea{
+  font: 400 13.3333px Arial;
```

Comment:
Hmm, wondering if we should actually prefer first some Unicode fonts (Arial Unicode,etc.) in preference order, and then `Arial` ?  Do popular Linux distros have them?  What about Mac?